blog-homepage/src/components/MissedArticle.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Author from './Author';
+import './Article.css';
+import Utils from './Utils';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBookmark, faStar, faVolumeUp } from '@fortawesome/free-solid-svg-icons';
+
+class MissedArticle extends React.Component {
+
+  static propTypes = {
+    article: PropTypes.exact({
+      title: PropTypes.string,
+      description: PropTypes.string,
+      image: PropTypes.string,
+      link: PropTypes.string,
+      author: PropTypes.exact({
+        name: PropTypes.string,
+        image: PropTypes.string,
+        isMediumMember: PropTypes.bool,
+      }),
+      postedDate: PropTypes.string,
+      minutesToRead: PropTypes.number,
+      hasAudioAvailable: PropTypes.bool,
+      memberPreview: PropTypes.bool,
+    }).isRequired,
+
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = { isBookmarkOn: false };
+    this.toggleBookmark = this.toggleBookmark.bind(this);
+  }
+
+  toggleBookmark() {
+    this.setState(state => ({
+      isBookmarkOn: !state.isBookmarkOn
+    }));
+    console.log('clicked');
+  }
+
+  render() {
+    const {article} = this.props;
+
+    return (
+      <article className="grid-top-image">
```

Comment:
A suggestion: You could also combine these two article components into one by passing in the classname as a prop.